lib/stories.js

```diff
@@ -73,8 +73,9 @@ const Wrapper = styled.div`
 
 const Nav = styled.div`
   position: sticky;
-  top: 10.5rem;
-  height: calc(100% - 10.5rem);
+  top: 1em;
+  height: calc(100vh - 1em);
+  overflow: scroll;
```

Comment:
should we use `overflow: auto` here instead of scroll so that the scrollbars aren't visible unless they're needed?